src/odemis/model/test/remote_test.py

```diff
@@ -300,7 +300,7 @@ def test_dataflow_unsub(self):
 
         comp2.sub(comp.data)
         time.sleep(0.5)
-#         comp2.unsub()
+        comp2.unsub()
```

Comment:
This should not be uncommented. It's to show where the unsub() call would be happening, but it's not happening and still things should work.... What you can do is to add a comment explaining that.